build.gradle.kts

```diff
@@ -17,6 +17,13 @@ java {
 
 dependencies {
     spotbugsPlugins("com.h3xstream.findsecbugs:findsecbugs-plugin:1.7.1")
+
+    compile("org.ahocorasick", "ahocorasick", "0.4.0")
```

Comment:
Versions should be in `gradle.properties`.

Reply:
I understand why we want to do that where we have multiple subprojects and want to share the same version between all projects, but in cases like this where we have a single gradle project what benefit do we get from this?

Reply:
We _will_ have multiple subprojects under this repository. Building several restful APIs is part of this product.

Besides that, it forces you to think about the current version you're using rather than copying the `Gradle` section from mvnrepository.com.

Reply:
Actually, I'd be inclined to drop this for something like `dependabot`

Reply:
(cc @danielgrant)

Reply:
Could use [Renovate](https://github.com/renovatebot/renovate), used quite well on other Frontend OSS. Can be configured to auto-merge patch versions so long as the tests pass.